webapp/app/css/logviewer.css

```diff
@@ -95,6 +97,7 @@ body {
     -o-user-select: text;
     user-select: text;
     white-space: normal;
+    font-size: 12px;
```

Comment:
We need to override here its .btn inheritance (14px), otherwise I would have set 12 only on .run-data.